docs/configuration.asciidoc

```diff
@@ -46,44 +48,42 @@ public class Startup
 }
 ----
 
-NOTE: As explained in <<setup-asp-net-core>>, the `UseElasticApm` method only turns on ASP.NET Core monitoring. If you want to turn on everything (including HTTP and Database monitoring) you can use the `UseAllElasticApm` method from the `Elastic.Apm.NetCoreAll` package, which has the same overloads and behavior, except it turns on tracing for everything that is supported by the agent on .NET Core including database and HTTP tracing.
-
 With this you can use any configuration source that you configured on the `IConfiguration` instance that you passed to the APM Agent.
-You can find the key of each configuration option
-in the `IConfiguration or Web.config key` column of the corresponding option's description.
+You can find the key for each APM configuration option in this documentation, under the `IConfiguration or Web.config key` column of the option's description.
+
+NOTE: It is also possible to call `UseElasticApm()` without the overload. In this case, the agent will only read configurations from environment variables.
 
-NOTE: By simply calling `app.UseElasticApm()` without the overload, the agent will read configurations only from environment variables.
+NOTE: The `UseElasticApm` method only turns on ASP.NET Core monitoring. To turn on tracing for everything supported by the Agent on .NET Core, including HTTP and database monitoring, use the `UseAllElasticApm` method from the `Elastic.Apm NetCoreAll` package. Learn more in <<setup-asp-net-core,ASP.NET Core setup>>.
```

Comment:
I'm not pumped on two NOTEs in a row, but I think they make more sense at the end of this section, instead of breaking it up like before.